### PR TITLE
Exercise list view update

### PR DESCRIPTION
**Pull request by Maijjay**

### CODE REVIEW

File: services/main-frontend/src/components/page-specific/manage/courses/id/exercises/CourseExercises.tsx [hunk 1]
```diff
@@ -18,9 +18,10 @@ const CourseExercises: React.FC<React.PropsWithChildren<CourseManagementPagesPro
       <h2
         className={css`
           font-size: clamp(2rem, 3.6vh, 36px);
-          color: ${baseTheme.colors.gray[700]};
+          color: ${baseTheme.colors.gray["700"]};
```
Comment (nygrenh) [inline services/main-frontend/src/components/page-specific/manage/courses/id/exercises/CourseExercises.tsx:21]: Not a string

File: services/main-frontend/src/components/page-specific/manage/courses/id/exercises/ExerciseList.tsx [hunk 2]
```diff
@@ -34,66 +39,164 @@ const ExerciseList: React.FC<React.PropsWithChildren<ExerciseListProps>> = ({ co
...
+                      </h3>
+                      <p
+                        className={css`
+                          font-weight: ${fontWeights.normal};
+                          font-size: 18px;
+                          line-height: 24px;
+
+                          color: ${baseTheme.colors.green[600]};
+                        `}
+                        // PAGE should be replaced with {t("page-number")}, once I know how
```
Comment (nygrenh) [inline services/main-frontend/src/components/page-specific/manage/courses/id/exercises/ExerciseList.tsx:110]: Outdated comment

File: services/main-frontend/src/components/page-specific/manage/courses/id/exercises/ExerciseList.tsx [hunk 3]
```diff
@@ -34,66 +39,164 @@ const ExerciseList: React.FC<React.PropsWithChildren<ExerciseListProps>> = ({ co
...
+                                href={{
+                                  pathname: "/manage/exercises/[exerciseId]/submissions",
+                                  query: { exerciseId: exercise.id },
+                                }}
+                              >
+                                {exercise.name} {exercise.count}
+                              </Link>
+                            </div>
+                            <div>
+                              {exercise.count === null ? (
```
Comment (nygrenh) [inline services/main-frontend/src/components/page-specific/manage/courses/id/exercises/ExerciseList.tsx:167]:
```
{exercise.count !== null && (<Link>..</Link>)}
```

File: shared-module/src/locales/en/main-frontend.json [hunk 4]
```diff
@@ -328,6 +328,7 @@
   "other-language": "other",
   "override-completion-registration-link": "Override completion registration link",
   "page-deletion-confirmation-message": "Are you sure you want to delete page {{title}}?",
+  "page-number": "PAGE {{page-number}}",
```
Comment (nygrenh) [inline shared-module/src/locales/en/main-frontend.json:331]: Write with normal case, and use `text-transform: uppercase` in the component where this is used.

**Review by nygrenh — CHANGES_REQUESTED** [review]

**Review by d471061c — APPROVED** [review]

**Review by nygrenh — APPROVED** [review]

**Pull request merged**